app/views/newsletters/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="newsletter-text">
+  <p>
+    Want to be kept up to date with the latest news from Any Rat Rescue? 
+  </p>
+  <button class="subscribe"><%= link_to "Subscribe", "#" %></button>
```

Comment:
Sorry, it looks like the button mixin from bourbon has been deprecated (see here: http://bourbon.io/docs/#buttons ). You could add [bitters](http://bitters.bourbon.io/) to the project and that will give you that button style. 

Usually I try and just use `link_to`s and just style them as buttons.